integrationtests.py

```diff
@@ -151,6 +151,16 @@ def testRecompileObjectSetOtherDir(self):
         subprocess.check_call(cmd) # Compile once
         subprocess.check_call(cmd) # Compile again
 
+    def testPipedOutput(self):
+        with cd(ASSETS_DIR):
+            cmd = [PYTHON_BINARY, CLCACHE_SCRIPT, "/nologo", "/c", "fibonacci.c", "/Fofibonacci_c.obj"]
+            try:
+                output = subprocess.check_output(cmd)
+            except subprocess.CalledProcessError as e:
+                self.fail('{}\nOutput:\n{}'.format(e, e.output))
+            self.assertNotEqual(output, b'')
+            self.assertTrue(b'\r\r\n' not in output, 'Output has duplicated CR: {}'.format(repr(output)))
```

Comment:
Looks like there is no `in` operation on binary data in Python 3.5. We can get around that by converting to string. This does not change the `\r`/`\n` situation:

```
output = subprocess.check_output(cmd).decode(clcache.CL_DEFAULT_CODEC)
```